cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1579,7 +1579,10 @@ def make_resource(self, data, **kwargs):
 class SlurmSettingsSchema(BaseSchema):
     """Represent the schema of the Scheduling Settings."""
 
-    scaledown_idletime = fields.Int(metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
+    scaledown_idletime = fields.Int(
```

Comment:
let's cover this case and a lower value with a unit test

Reply:
done